app/javascript/gobierto_dashboards/webapp/containers/contracts/Home.vue

```diff
@@ -65,6 +65,11 @@ export default {
     EventBus.$on('refresh-summary-data', () => {
       this.contractsData = this.$root.$data.contractsData;
     });
+    EventBus.$on("update-tab", () => {
+      // eslint-disable-next-line no-unused-vars
+      this.$router.push('adjudicaciones').catch(err => {})
```

Comment:
Actually you don't need the first sentence if you take off the `catch`

Reply:
This is to avoid an error. When you go to the same URL you got an error `Throw Error "NavigationDuplicated"`

https://github.com/vuejs/vue-router/issues/2872